bitnami/logstash/values.yaml

```diff
@@ -0,0 +1,380 @@
+  #   secretName: logstash.local-tls
+
+  secrets:
+  ## If you're providing your own certificates, please use this to add the certificates as secrets
+  ## key and certificate should start with -----BEGIN CERTIFICATE----- or
+  ## -----BEGIN RSA PRIVATE KEY-----
+  ##
+  ## name should line up with a tlsSecret set further up
+  ## If you're using cert-manager, this is unneeded, as it will create the secret for you if it is not set
+  ##
+  ## It is also possible to create and manage the certificates outside of this helm chart
+  ## Please see README.md for more information
+  # - name: logstash.local-tls
+  #   key:
+  #   certificate:
+
+## Prometheus metrics
+##
+metrics:
+  enabled: false
+  ## Bitnami Logstash Promtheus Exporter image
+  ## ref: https://hub.docker.com/r/bitnami/logstash-exporter/tags/
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/logstash-exporter
+    tag: 0.1.2-debian-9-r0
+    ## Specify a imagePullPolicy. Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets (secrets must be manually created in the namespace)
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ##
+    # pullSecrets:
+    #   - myRegistryKeySecretName
+
+  ## Logstash Prometheus Exporter containers' resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    limits: {}
+    #   cpu: 100m
+    #   memory: 128Mi
+    requests: {}
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  ## Logstash Prometheus Exporter containers' liveness and readiness probes
+  ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  ##
+  livenessProbe:
+    httpGet:
+      path: /metrics
+      port: metrics
+    initialDelaySeconds: 60
+    # periodSeconds: 10
+    # timeoutSeconds: 5
+    # successThreshold: 1
+    # failureThreshold: 6
+  readinessProbe:
+    httpGet:
+      path: /metrics
+      port: metrics
+    initialDelaySeconds: 60
+    # periodSeconds: 10
+    # timeoutSeconds: 5
+    # successThreshold: 1
+    # failureThreshold: 6
+
+  service:
+    ## Service type
+    ##
+    type: ClusterIP
+    ## Logstash Prometheus port
+    ##
+    port: 9198
+    ## Specify the nodePort value for the LoadBalancer and NodePort service types.
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+    ##
+    nodePort: ""
```

Comment:
You are not allowing nodePort in the normal service

Reply:
The ports defined in the "normal" services are totally customizable. Therefore, you can add the `nodePort` there